apis/feedapi/feedapi.yaml

```diff
@@ -0,0 +1,117 @@
+swagger: '2.0'
+info:
+  version: '1.0'
+  title: User Feed  API
+  description: >-
+    - The User Feed APIs  are used to store all alerts that could be presented to users. They include alerts sent to users while migrating tenant environments from one channel to another.
+host: staging.open-sunbird.org
+basePath: /api/
+securityDefinitions:
+  bearer:
+    type: apiKey  # The arbitrary value for the security scheme. This is provided to the user, when the user sends a request for the API key. It is a concatenated string.   
```

Comment:
@anmol2302 please check the way this sentence is rephrased, based on the explanation given.

Reply:
@Sunitashelar yes it was correct